### DISCUSSION TITLE
V1.7 doesn't enable HDR when launching game

### DISCUSSION THREAD

**Discussion by rb2k5** [Codectory/AutoActions#65] [Q&A] [answered]
What am I doing wrong? Since the release of v1.7 I haven't been able to get the app to automatically switch my screen to HDR mode when launching an added app. The activate HDR toggle button works fine, but the auto-switching doesn't work.
I used 1.6 for many months without issue, and going back to 1.6 works fine.

**Comment by Codectory** [comment]
could you please send me a log file and the UserSettings.json?

**Comment by rb2k5** [comment, marked as answer]
Ignore me. I'm an idiot.
I hadn't created a profile or assigned it to the apps.
All good now.
